### DISCUSSION TITLE
Font not applying correctly in Kitty terminal

### DISCUSSION THREAD

**Discussion by kadeguilherme** [kovidgoyal/kitty#8089] [General]
Description:

I'm trying to set the JetBrains Mono font in my Kitty terminal, but it's not being applied correctly. Despite configuring it in the kitty.conf file, the font is not showing up as expected. I’ve checked the font installation and it seems to be correctly installed on my system.

Here’s my configuration in kitty.conf:
![image](https://github.com/user-attachments/assets/55a73067-9fb2-41ea-b70b-2e2256ae0564)

```
font_family JetBrains Mono
bold_font auto
italic_font auto
bold_italic_font auto

```

Any help would be appreciated to figure out why the font isn’t being applied properly. Thanks!

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font
